Content.Server/Forensics/Systems/ForensicsSystem.cs

```diff
@@ -0,0 +1,72 @@
+using System.Linq;
+using System.Text;
+using Content.Server.Hands.Systems;
+using Content.Shared.Hands.Components;
+using Content.Shared.Interaction;
+using Content.Shared.Interaction.Events;
+using Content.Shared.Inventory;
+using Content.Shared.Item;
+using Robust.Shared.Containers;
+using Robust.Shared.Random;
+
+namespace Content.Server.Forensics
+{
+    public sealed class ForensicsSystem : EntitySystem
+    {
+        [Dependency] private readonly IRobustRandom _random = default!;
+        [Dependency] private readonly InventorySystem _inventory = default!;
+        [Dependency] private readonly HandsSystem _handsSystem = default!;
+
+        public override void Initialize()
+        {
+            SubscribeLocalEvent<ForensicsComponent, InteractHandEvent>((uid, component, args) => ApplyEvidence(args.User, component));
+            SubscribeLocalEvent<ForensicsComponent, GettingInteractedWithAttemptEvent>((uid, component, args) =>
+            {
+                Hand hand = _handsSystem.EnumerateHands(args.Uid).First();
+                if (HasComp<ForensicScannerComponent>(hand.HeldEntity))
+                    return;
+                ApplyEvidence(args.Uid, component);
+            });
```

Comment:
this is backwards, it should really be `SubscribeLocalEvent<ForensicScannerComponent, AfterInteractEvent>` (which I believe you already have a subscription for, so just do it in there instead of doing this weird hand stuff). 

either way the attempt events shouldnt be used for this since they're just 'attempt' events, theres no guarantee it'll actually work